readme.md

```diff
@@ -30,8 +31,9 @@ Make all the changes smooth on hover (during 300ms):
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+
+- [DEMO LINK](https://seriuksergii/.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://seriuksergii/.github.io/layout_catalog/report/html_report/)
```

Comment:
I found your demo and tests ) extra slash in the link.
Do not forget to add these links in the description of the pull request.

```suggestion
- [DEMO LINK](https://seriuksergii.github.io/layout_catalog/)
- [TEST REPORT LINK](https://seriuksergii.github.io/layout_catalog/report/html_report/)
```